packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -111,10 +118,11 @@ export class DataStores implements IDisposable {
                         snapshotTree.blobs,
                         snapshotTree.blobs[".component"]);
                 // Use the snapshotFormatVersion to determine how the pkg is encoded in the snapshot.
-                // For snapshotFormatVersion = "0.1", pkg is jsonified, otherwise it is just a string.
+                // For snapshotFormatVersion = "0.1" or "0.2", pkg is jsonified, otherwise it is just a string.
                 // However the feature of loading a detached container from snapshot, is added when the
                 // snapshotFormatVersion is "0.1", so we don't expect it to be anything else.
-                if (snapshotFormatVersion === currentSnapshotFormatVersion) {
+                if (snapshotFormatVersion === dataStoreSnapshotFormatVersions.current
```

Comment:
I'm a bit confused here. Existing files will take the else branch, no?

Reply:
That's intentional based on the comment:
> // However the feature of loading a detached container from snapshot, is added when the
// snapshotFormatVersion is "0.1", so we don't expect it to be anything else.